src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -84,6 +85,10 @@ public void displayResult(CommandResult result) {
         if(resultPersons.isPresent()) {
             display(resultPersons.get());
         }
+        final Optional<ArrayList<String>> resultCommands = result.getRelevantCommands();
+        if(resultCommands.isPresent()) {
```

Comment:
LOL... I think I did spot this issue occasionally in their code base. Nvm it's okay this address book thing does not need to be perfect anw lol